shared/rangeanalysis/codeql/rangeanalysis/RangeAnalysis.qll

```diff
@@ -296,11 +363,26 @@ signature module LangSig<Semantic Sem, DeltaSig D> {
 }
 
 signature module BoundSig<LocationSig Location, Semantic Sem, DeltaSig D> {
+  /**
+   * A semantic bound, which defines a constraint on the possible values of an
+   * expression.
+   */
   class SemBound {
+    /**
+     * Gets a string representation of the semantic bound.
+     */
     string toString();
 
+    /**
+     * Gets the location of the semantic bound.
+     */
     Location getLocation();
 
+    /**
+     * Gets the expression associated with the semantic bound, given a delta.
+     *
+     * @param delta - The delta value.
+     */
```

Comment:
I tried to get Copilot to tell me what the delta is all about, it pretty much admitted that it didn't know for sure.

Reply:
Addressed.